devops/e2e/cloudtest/ubuntu-22_04.json

```diff
@@ -0,0 +1,31 @@
+{
+    "artifacts": [
+        {
+            "name": "linux-add-key",
+            "parameters": {
+                "key": "https://packages.microsoft.com/keys/microsoft.asc"
+            }
+        },
+        {
+            "name": "linux-add-repository",
+            "parameters": {
+                "repository": "https://packages.microsoft.com/ubuntu/22.04/prod"
+            }
+        },
+        {
+            "name": "linux-install-packages",
+            "parameters": {
+                "packages": "aziot-identity-service jq"
+            }
+        },
```

Comment:
Why are we instrumenting Ubuntu 22.04 as it were to support AIS? Does AIS work here and does not work in the other case? Please explain.

Reply:
It does get installed on the VM, however the service does not appear to be available/OSConfig is unable to connect to it and obtain a the connection string. I was able to confirm that the connection string is added to the AIS `config.toml` file, however the socket that the PnP agent uses to communicate with AIS is not available.

Reply:
Then we should skip AIS for good and do not install the package on Ubuntu 22.04 as it won't be used anyway?